doc/nrf/working_with_nrf/nrf52/gs.rst

```diff
@@ -60,26 +60,31 @@ The steps detailed in :ref:`nrf52_gs_connecting` use PuTTY, but any terminal emu
 
 On your mobile device, install the `nRF Connect for Mobile`_ mobile application from the corresponding application store.
 
+.. _nrf52_gs_installing_sample:
 .. _nrf52_gs_installing_application:
 
-Installing the application
-**************************
+Installing the sample
+*********************
 
-You must program and run a precompiled version of the :ref:`peripheral_uart` sample application on your development kit to test the functions.
+You must program and run a precompiled version of the :ref:`peripheral_uart` sample on your development kit to test the functions.
 
-Download the precompiled version of the application for your DK from the corresponding download page:
+Download the precompiled version of the sample for your DK from the corresponding download page:
 
 * `nRF52840 DK Downloads`_
 * `nRF52833 DK Downloads`_
 * `nRF52 DK Downloads`_
 
 After downloading the zip archive, extract it to a folder of your choice.
-The archive contains the HEX file used to program the application to your DK.
+The archive contains the HEX file used to program the sample to your DK.
 
-To program the precompiled sample application to your development kit, complete the following steps:
+.. |DK| replace:: nRF52 Series DK
+
+.. program_dk_sample_start
+
+To program the precompiled sample sample to your development kit, complete the following steps:
```

Comment:
```suggestion
To program the precompiled sample to your development kit, complete the following steps:
```
Looks like there's an accidental duplicate here.

Reply:
Good catch, used to say sample application and did the replacement too quickly.